src/chat/Chat.js

```diff
@@ -1,40 +1,25 @@
 import React from 'react';
-import {
-  KeyboardAvoidingView,
-} from 'react-native';
+import { KeyboardAvoidingView } from 'react-native';
 
 import { styles, OfflineNotice } from '../common';
 import { canSendToNarrow } from '../utils/narrow';
-import MessageList from '../message-list/MessageList';
+import MessageList from '../message/MessageList';
+import MessageListLoading from '../message/MessageListLoading';
 import NoMessages from '../message/NoMessages';
 import ComposeBox from '../compose/ComposeBox';
 
 export default class MainScreen extends React.Component {
-
   render() {
-    const { auth, messages, narrow, fetching, caughtUp, subscriptions,
-      isOnline, twentyFourHourTime, doNarrow, fetchOlder, fetchNewer,
-      markAsRead, mute } = this.props;
+    const { messages, narrow, caughtUp, fetching, isOnline } = this.props;
```

Comment:
Can we get rid of `Chat.js` too and just use `MessageList` from `MainScreen`? I feel like it adds an unnecessary level of indirection